department-of-veterans-affairs/abd-vro
# Format code consistently using pre-commit

## What was the problem?


Files were inconsistently formatted in terms of end-of-file and trailing-whitespace.
Due to the new `pre-commit` hook (#659), folks will have to address them on the next modification of a non-conforming file.

## How does this fix it?


Fix most of the pre-commit alerts in this PR to reduce the burden on developers.

## Tip for reviewing this PR
Enable "Hide whitespace" when reviewing to see that only formatting has been changed for all files except for file `/build.gradle`:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/55255674/200372230-91dcef2c-e2a9-4835-8778-fdc2c09bf725.png">

## How to test this PR
Manually test VRO to ensure functionality has NOT changed.

To see other fixes that can be done, run `pre-commit run --all-files`. Only Python files remain to be updated.
